### ISSUE TITLE
Could not find KotlinAudio v0.1.29

### ISSUE THREAD

**Issue by ismailassa** [doublesymmetry/KotlinAudio#74]
KotlinAudio v0.1.29 could not be found when building for android, see the error message bellow:
The https links return a 404 page
```
Could not resolve all files for configuration ':app:[REDACTED]RuntimeClasspath'.
Could not find com.github.DoubleSymmetry:KotlinAudio:v0.1.29.
Searched in the following locations:
- https://repo.maven.apache.org/maven2/com/github/DoubleSymmetry/KotlinAudio/v0.1.29/KotlinAudio-v0.1.29.pom
- file:/Users/vagrant/.m2/repository/com/github/DoubleSymmetry/KotlinAudio/v0.1.29/KotlinAudio-v0.1.29.pom
- file:/Users/vagrant/git/node_modules/react-native/android/com/github/DoubleSymmetry/KotlinAudio/v0.1.29/KotlinAudio-v0.1.29.pom
- https://dl.google.com/dl/android/maven2/com/github/DoubleSymmetry/KotlinAudio/v0.1.29/KotlinAudio-v0.1.29.pom
- https://www.jitpack.io/com/github/DoubleSymmetry/KotlinAudio/v0.1.29/KotlinAudio-v0.1.29.pom
Required by:
project :app > project :react-native-track-player
```

**Comment by xenokratos** [comment]
Triggering the build on `jitpack` did fix it -> [KotlinAudio - v0.1.29](https://jitpack.io/#DoubleSymmetry/KotlinAudio/v0.1.29)
![Screenshot 2023-04-04 at 13 46 12](https://user-images.githubusercontent.com/46603154/229783109-30e81357-4915-4450-93f5-359ed4f967e8.png)

It works but still, having a do it manually is not optimal 😅

**Comment by jspizziri** [comment]
@dcvz I believe this issue can be closed.
